packages/ssi-types/src/types/w3c-vc.ts

```diff
@@ -278,6 +278,7 @@ export interface IErrorDetails {
 
 export enum StatusListType {
   StatusList2021 = 'StatusList2021',
+  OAuth2StatusList = 'OAuth2StatusList',
```

Comment:
This should not end up in w3c-vc for now. OAuth is not compatible

Reply:
fixed